### PR TITLE
feat: Add v1 Alert Notificaiton Target validation

### PR DESCRIPTION
**Pull request by nieomylnieja**

### CODE REVIEW

**Comment by nieomylnieja** [comment]
Branched off of https://github.com/OpenSLO/OpenSLO/pull/295, once this PR is merged I will rebase to main.

File: pkg/openslo/v1/alert_notification_example_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
+package v1_test
+
+import (
+	"bytes"
+	"os"
+	"reflect"
+
+	v1 "github.com/OpenSLO/OpenSLO/pkg/openslo/v1"
```
Comment (fpiwowarczyk) [inline pkg/openslo/v1/alert_notification_example_test.go:8]: It does nothing
```suggestion
	"github.com/OpenSLO/OpenSLO/pkg/openslo/v1"
```

**Review by fpiwowarczyk — COMMENTED** [review]
One *nit* and I'll approve. Great work!

File: pkg/openslo/v1/alert_notification_example_test.go (hunk 1 above)
Comment (fpiwowarczyk) [reply pkg/openslo/v1/alert_notification_example_test.go:8]: Ok turns out it is intended behaviour of goimports to add this alias lol

**Review by fpiwowarczyk — APPROVED** [review]
lgtm

**Pull request merged**